src/frontend/src/app/asset-readings/assets/assets.component.html

```diff
@@ -59,7 +59,7 @@ <h5 class="title"><small> Asset Readings </small></h5>
           <thead>
             <tr>
               <th>Asset</th>
-              <th>Readings&nbsp;&nbsp;<i class="fa fa-info-circle tooltip is-tooltip-right" data-tooltip="DD-MM-YYYY HH:mm:ss"></i></th>
+              <th>Readings&nbsp;&nbsp;<span class="tooltip is-tooltip-right" data-tooltip="DD-MM-YYYY HH:mm:ss"><i class="fa fa-info-circle"></i></span></th>
```

Comment:
This issue should also be fix in `list-tasks.component.html`

Reply:
done!